code/game/gamemodes/objective.dm

```diff
@@ -311,7 +311,7 @@ var/list/potential_theft_objectives=subtypesof(/datum/theft_objective) \
 		return 0
 
 /datum/objective/block
-	explanation_text = "Do not allow any organic lifeforms to escape on the shuttle alive."
+	explanation_text = "Do not allow any organic lifeforms or IPCs to escape on the shuttle alive."

```

Comment:
"Do not allow any lifeforms, be it organic or synthetic to escape on the shuttle alive." would sound better. Just saying.